packages/runtime/README.md

```diff
@@ -1,12 +1,46 @@
 # Runtime
 
-JavaScript Algorand runtime. It provides:
+This package implements a light version of Algorand runtime. It allows to executes transactions and processes TEAL in JavaScript environment. The packages provides 4 main object:
 
-- Account state
-- TEAL stateless interpreter
-- TEAL stateful interpreter
+- [Runtime](../packages/runtime/src/runtime.ts): handles transaction or txn group processing and state management . User can use a `Runtime` object to set up accounts, create applications, opt-in to app, update app, etc...
```

Comment:
Please update the link here (`../packages/runtime/src/*` -> `./src/*`)

Reply:
Ok